src/components/SpiceList.js

```diff
@@ -8,6 +8,14 @@ class SpiceList extends React.Component {
     search: ""
   }
 
+  // Takes in search term and changes state.search to that search term; to be passed down to Filter child
+  changeSearch = searchTerm => this.setState({ search: searchTerm })
+  
+
+  // Will change the fourStarOnly to the boolean value passed; to be passed down to Filter child
+  changeFourStar = bool => this.setState({ fourStarOnly: bool });
```

Comment:
Nice! One trick here you could try for shortening the syntax would be to use [shorthand property names](https://ui.dev/shorthand-properties/#:~:text=With%20Shorthand%20Properties%2C%20whenever%20you,can%20omit%20the%20property%20name.) like this:

```js
handleSearchState = search => this.setState({ search })

handleCheckedState = fourStarOnly => this.setState({ fourStarOnly })
```

This works as long as the key in state matches the name of the variable you're trying to assign to it.